toplevel/coqargs.ml

```diff
@@ -459,7 +459,7 @@ let parse_args ~help ~init arglist : t * string list =
       { oval with batch = true }
     |"-test-mode" -> Flags.test_mode := true; oval
     |"-beautify" -> Flags.beautify := true; oval
-    |"-boot" -> { oval with boot = true; load_rcfile = false; }
+    |"-boot" -> warn_deprecated_boot (); oval
```

Comment:
Shouldn't we still set -q while in the deprecation phase?

Reply:
Ok, why not?

Reply:
Resolved.